Sources/_CryptoExtras/RSA/RSA.swift

```diff
@@ -132,6 +142,11 @@ extension _RSA.Signing {
         fileprivate init(_ backing: BackingPublicKey) {
             self.backing = backing
         }
+
+        public func getKeyPrimitives() throws -> Primitives {
```

Comment:
I'm inclined to want to make these effectful computed properties. What do you think @ptoffy?

Reply:
I'm honestly a bit torn 😅 having `key.modulus` and `key.publicExponent` does look cleaner, but if you mean having it as `key.primitives` we might as well leave it as a function: it throws, there's a conversion (on Darwin) and we _are_ calling BoringSSL so there _is_ some underlying computation. Odds are it won't get accessed often so caching doesn't really make sense and I'd say it's unlikely someone's accessing either one without the other(?)